src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -62,7 +62,6 @@
 
         <service id="messenger.transport.amqp.factory" class="Symfony\Component\Messenger\Transport\AmqpExt\AmqpTransportFactory">
             <argument type="service" id="messenger.transport.encoder" />
```

Comment:
This probably need to be changed to be `messenger.transport.serializer`.

Reply:
indeed

Reply:
Updated.